File: Unidad 2-Estructuras de Control/Ejercicio49.c
//Autores: Programa realizado por el Equipo2_CodePain
//Version 1.1
//Programa que imprime la tabla de multiplicar del 2 al 9

#include <stdio.h>
#include <math.h>

int main (){

//Entrada: Declaracion de variables
    int prod = 0, x= 0, y = 0;

//Proceso: Inicio de ciclos "for" para imprimir las tablas de multiplicar solicitadas
//Salida: Las tablas de multiplicar del 2 al 9
    for (x = 2; x <=10; x++ ){
        for (y = 1; y <=10; y++ ){
            prod = x * y ;

            printf("%d\n",prod);
        
    }
    }
    return 0;
}

File: Unidad 2-Estructuras de Control/Equipo2_CodePain/Ejercicio49.py
#Declaración de variables
prod=0 #Producto de la tabla de multiplicar 
x = 0 #Número que va de 2 al 10 
y = 0


#Inicio de ciclos for para imprimir las tablas de multiplicar solicitadas
#Salida: Las tablas de multiplicar del 2 al 9
for x in range(2,10):
    for y in range(1,11):
        prod = x * y
        print(str(x) + "x" + str(y) + "=" + str(prod)) 
